x-pack/test/security_solution_cypress/cypress/e2e/exceptions/shared_exception_lists_management/shared_exception_list_page/duplicate_lists.cy.ts

```diff
@@ -64,9 +64,7 @@ describe('Duplicate List', { tags: ['@ess', '@serverless'] }, () => {
     );
 
     // Create exception list not used by any rules
-    createExceptionList(getExceptionList1(), getExceptionList1().list_id).as(
-      'exceptionListResponse'
-    );
+    createExceptionList(getExceptionList1(), getExceptionList1().list_id);
```

Comment:
Why is this (an the other one below) change required?

Reply:
@dasansol92 Looks like the network request was actually not waited on anywhere in the test suite. Might have been leftover/redundant code from elastic/kibana/pull/152757